### PR TITLE
Refs 3419: Add ouiaIDs for template form

### PR DESCRIPTION
**Pull request by swadeley**
## Summary

I need some ouiaIDs in the content template form
Will try use the same as those in custom repo modal first.

## Testing steps

I'll test manually.

### CODE REVIEW

**Comment by swadeley** [comment]
Hi @Andrewgdewar , this is failing to build. Is it just the location or is code support required?

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

File: src/Pages/Templates/TemplatesTable/components/AddTemplate/steps/DefineContentStep.tsx [hunk 1]
```diff
@@ -71,6 +71,7 @@ export default function DefineContentStep() {
           setDisabled
         >
           <DropdownMenu
+	    ouiaId='restrict_to_architecture'
```
Comment (xbhouse) [inline src/Pages/Templates/TemplatesTable/components/AddTemplate/steps/DefineContentStep.tsx:74]: i think you may be able to do: `dropDownProps={{ouiaId: 'restrict_to_architecture'}}`, but maybe @adewar has a better way to add this that's more consistent with the other components?

Comment (Andrewgdewar) [reply src/Pages/Templates/TemplatesTable/components/AddTemplate/steps/DefineContentStep.tsx:74]: Yup, unsurprisingly @xbhouse  is correct.

Comment (swadeley) [reply src/Pages/Templates/TemplatesTable/components/AddTemplate/steps/DefineContentStep.tsx:74]: Thank you. I'll try add a few more.

Comment (swadeley) [reply src/Pages/Templates/TemplatesTable/components/AddTemplate/steps/DefineContentStep.tsx]: @Andrewgdewar not working yet

Comment (swadeley) [reply src/Pages/Templates/TemplatesTable/components/AddTemplate/steps/DefineContentStep.tsx]: Working now with 
menuToggleProps={{ 'data-ouia-component-id': 'restrict_to_os_version' }}
and 
menuToggleProps?: Partial<MenuToggleProps | unknown>;

**Review by Andrewgdewar — APPROVED** [review]

**Pull request merged**